index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="flex-container outer-container">
+        <header id="navbar" class="flex-container">
+            <!--- Item to display logo -->
+            <div class="flex-item" id="logo">
+                Welcome
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">Portfolio</a>
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">About</a>
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">Contact</a>
+            </div>
+            <div class="flex-item" id="link">
+                <i class="fa-solid fa-bars"></i></i>
+            </div>
+        </header>
+        <section class="flex-container main-container">
+                <div><h1 class="intro">Hey There. I'm Ahmad.</h1></div>
+                <div></div><h1 class="profession">I'm a Full-stack Web Developer.</h1></div>
+                <div><p>I can help you build interactive and responsive websites. To be sure about the quality of my work, you may view my portfolio.</p></div>
+        </section>
+        <section class="lower-container social-links">
```

Comment:
- [OPTIONAL] Kindly consider laying out and centering your icons with flex as in the figma design. you can give `display: flex` to this section and then try `justify_content: center` to center the ul properly and then apply same in the ul if needed to center the `icons`.
your design
![vc](https://user-images.githubusercontent.com/96683843/219464243-cb2b7aa4-d224-4d64-80ad-46660685956e.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/219464271-373576d7-c255-421e-bb83-cc31a658029e.jpg)